client/app/email-domain/delegate/email-domain-delegate.controller.js

```diff
@@ -114,6 +114,13 @@ angular.module("App").controller(
                     email.quota = usage.quota;
                     email.emailCount = usage.emailCount;
                     email.date = usage.date;
+
+                    if (email.size > 0) {
```

Comment:
Apparently this code is already present in the controller, maybe you would extract this bit of code in a function so you don't repeat yourself : https://en.wikipedia.org/wiki/Don%27t_repeat_yourself :smiley: